comparative_benchmark/benchmark_lib.py

```diff
@@ -54,15 +57,15 @@ def _run(
       "inputs": input_dims,
       "outputs": output_dims,
       "compiler": compiler,
-      "device": benchmark.target_device.name,
+      "device": target_device.name,
```

Comment:
@beckerhe FYI, with this change, the benchmark name no longer includes `.../target_devices/...` at the end. But we still populate the target device name in the `device` field of benchmark results.